src/NuGet.Core/NuGet.PackageManagement/IDE/ISolutionManager.cs

```diff
@@ -18,6 +18,8 @@ public interface ISolutionManager
 
         event EventHandler SolutionClosed;
 
+        event EventHandler AfterNuGetCacheUpdated;
```

Comment:
Discussed offline. Moving it to IVsSolutionmanager would require chnages to NuGetUIContext and that would require changes to anyone that consumes it.